### ISSUE TITLE
Escaping '/' for search engine completion

### ISSUE THREAD

**Issue by rperryng** [philc/vimium#3130]
I find that using [searching engine completion](https://github.com/philc/vimium/wiki/Search-Completion) is a very convenient way of building shortcuts for long URLs, e.g. to visit a specific subreddit I could do

```
r: https://www.reddit.com/r/%s Reddit
```

and type `r games` in the omnibar to visit the games subreddit.

Sometimes, there are URLs in which I would like to use a `/` token, e.g.

```
gh: https://www.github.com/%s
```
so that I could do `g pulls` to visit my pulls, but also `g rperryng/dotfiles` to visit a repo within an org/user.  However when trying this the string is URIencoded into `rperryng%2Fpulls`.

It seems like this [functionality is built in](https://github.com/philc/vimium/blob/17fb86e02971948f90215934cbe5417674364a29/lib/utils.coffee#L133), in some way, or at least implemented internally but I couldn't understand from the wiki pages or code if this can be achieved via the options.  Any help appreciated, thanks!

**Comment by smblott-github** [comment]
So, you're saying that we should only apply `encodeURIComponent` if the query contains non-URI characters?  That sounds reasonable.

**Comment by gdh1995** [comment]
I suggest that we use `%S` (upper-case 'S') to represent a "raw" query and not encode it, and keep `%s` to be URI-encoded.

**Comment by gdh1995** [comment]
Oops, I find an old issue #2580 which is also about this.

**Comment by smblott-github** [comment]
Fixed in #3136.
